tests/TestData/CPACS_20_D250_10.xml

```diff
@@ -6094,6 +6094,7 @@
 									</lowerShell>
 									<ribsDefinitions>
 										<ribsDefinition uID="rib_winglet_1">
+											<name/>
```

Comment:
Is this change required? What happens, if I load "old" files without a name element?